examples/streamz/python/cybert.py

```diff
@@ -0,0 +1,46 @@
+import cudf
```

Comment:
do you think providing flexibility to pass arguments apart from default values to the script would be nice?

Reply:
This might not be a bad idea.